client/src/test/RestaurantList.test.jsx

```diff
@@ -0,0 +1,48 @@
+import { configure, mount, shallow } from 'enzyme/build';
+import { RestaurantList, RestaurantListContainer } from '../RestaurantList/RestaurantList';
+import { Provider } from 'react-redux';
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import Adapter from 'enzyme-adapter-react-16/build';
+import { FETCH_RESTAURANTS_REQUEST, mockRestaurants } from '../consts';
+
+configure({ adapter: new Adapter() });
+
+describe('RestaurantList', () => {
+  let store;
+  let fetchRestaurants;
+
+  beforeEach(() => {
+    const mockStore = configureMockStore();
```

Comment:
you only need this line once. The next line is the one you want to repeat (if any.)